Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.scss

```diff
@@ -60,5 +60,9 @@
     display: flex;
     align-items: center;
     justify-content: center;
+
+    svg {
+      width: 100%;
+    }
```

Comment:
Please check before merging if we really need this

Reply:
Yep, this is a problem in with Edge 42.17134.1.0 (Microsoft EdgeHTML 17.17134)